### ISSUE TITLE
add text labels to WCAG guideline success criteria

### ISSUE THREAD

**Issue by jshslsky** [ucla-oarc-mobile/casa-on-rails#100]
Currently only the numeric label is shown which is poor UX. The label should be added to the edit and show functions.

**Comment by jshslsky** [comment]
Now in production.
